_Group Projects/Group_5/README.Rmd

```diff
@@ -0,0 +1,17 @@
+---
+title: "README"
+output: md_document
+date: "`r Sys.Date()`"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+The program group5 loads an NLS dataset and, upon the specification of two variables, performs a regression and exports the results in a .docx format. To use the program, the user would specify two variables of choice: group5 [var1] [var2]
```

Comment:
A few ideas of what else to specify (that I'm sure other reviewers can add to):

- How you handle missing values
- What kind of data (numerical, categorical, etc.) can be included as inputs
- Examples of output

Reply:
I agree with Sylvia's comments! A bit more specificity in the README file would be great!

Reply:
I have endeavored to address all of sbrown5x's comments. Please let me know if you have any other helpful suggestions.